testproject-tools-integration/Packages/manifest.json

```diff
@@ -1,10 +1,11 @@
 {
   "registry": "https://artifactory.prd.cds.internal.unity3d.com/artifactory/api/npm/upm-candidates",
   "dependencies": {
-    "com.unity.ide.rider": "3.0.7",
-    "com.unity.netcode.gameobjects": "file:../../com.unity.netcode.gameobjects",
-    "com.unity.multiplayer.tools": "1.0.0-pre.4",
+    "com.unity.ide.rider": "3.0.9",
+    "com.unity.ide.visualstudio": "2.0.14",
+    "com.unity.multiplayer.tools": "1.0.0-pre.5",
     "com.unity.netcode.adapter.utp": "file:../../com.unity.netcode.adapter.utp",
+    "com.unity.netcode.gameobjects": "file:../../com.unity.netcode.gameobjects",
```

Comment:
we probably don't need changes in `manifest.json`

Reply:
Yes we do...
Take note of this:  "com.unity.multiplayer.tools": "1.0.0-pre.5"

Reply:
so, we only need `"com.unity.multiplayer.tools": "1.0.0-pre.4"` to `"com.unity.multiplayer.tools": "1.0.0-pre.5"` change then? :)

Reply:
The only other change was keeping the "com.unity.ide.visualstudio": "2.0.14" in place so anyone using visual studio wouldn't need to have to keep adding the package.  I already got approval from Beck on this addition, but if you think it should be removed then I will remove it.